packages/visualizer/src/uiSetup.js

```diff
@@ -0,0 +1,118 @@
+
+/**
+ *
+ * @param domElement
+ * @param layers
+ * @param itownsView
+ * @param options
+ */
+export function setupLoadingUI(domElement, layers, itownsView, options) {
+  const c3DTilesLoadingDomElement = document.createElement('div');
+  if (options.c3DTilesLoadingDomElementClasses) {
+    options.c3DTilesLoadingDomElementClasses.forEach((cssClass) => {
+      c3DTilesLoadingDomElement.classList.add(cssClass);
+    });
+  }
+  domElement.appendChild(c3DTilesLoadingDomElement);
+
+  const camera3D = itownsView.camera.camera3D;
+  c3DTilesLoadingDomElement.hidden = true;
+
+  /** @type {Map<string, Mesh>} */
+  const currentLoadingBox = new Map();
+
+  const loadingBoxId = (layer, tileId) => layer.id + tileId;
+
+  layers.forEach((c3dTilesLayer) => {
+    c3dTilesLayer.addEventListener(
+      C3DTILES_LAYER_EVENTS.ON_TILE_REQUESTED,
+      ({ metadata }) => {
+        if (metadata.tileId == undefined) throw new Error('no tile id');
+
+        const worldBox3 = metadata.boundingVolume.box.clone();
+
+        if (metadata.transform) {
+          worldBox3.applyMatrix4(metadata.transform);
+        } else if (metadata._worldFromLocalTransform) {
+          worldBox3.applyMatrix4(metadata._worldFromLocalTransform);
+        }
+
+        const box3Object = new Mesh(
+          new BoxGeometry(),
+          new MeshBasicMaterial({
+            wireframe: true,
+            wireframeLinewidth: 2,
+            color: new Color(Math.random(), Math.random(), Math.random()),
+          })
+        );
+        box3Object.scale.copy(worldBox3.max.clone().sub(worldBox3.min));
+        worldBox3.getCenter(box3Object.position);
+        box3Object.updateMatrixWorld();
+        itownsView.scene.add(box3Object);
+        itownsView.notifyChange(camera3D);
+
+        // bufferize
+        currentLoadingBox.set(
+          loadingBoxId(c3dTilesLayer, metadata.tileId),
+          box3Object
+        );
+        c3DTilesLoadingDomElement.hidden = false;
+      }
+    );
+
+    c3dTilesLayer.addEventListener(
+      C3DTILES_LAYER_EVENTS.ON_TILE_CONTENT_LOADED,
+      ({ tileContent }) => {
+        if (
+          currentLoadingBox.has(loadingBoxId(c3dTilesLayer, tileContent.tileId))
+        ) {
+          itownsView.scene.remove(
+            currentLoadingBox.get(
+              loadingBoxId(c3dTilesLayer, tileContent.tileId)
+            )
+          );
+          currentLoadingBox.delete(
+            loadingBoxId(c3dTilesLayer, tileContent.tileId)
+          );
+          c3DTilesLoadingDomElement.hidden = currentLoadingBox.size == 0; // nothing more is loaded
+          itownsView.notifyChange(camera3D);
+        }
+      }
+    );
+  });
+  return c3DTilesLoadingDomElement;
+}
+
+/**
+ *
+ * @param orbitControls
+ */
+export function setUpSpeedControls(orbitControls) {
+  const domElementSpeedControls = document.createElement('div');
+  const sliderSpeedControls = createLocalStorageSlider(
+    'speed_orbit_controls',
+    'Controls vitesse',
```

Comment:
`controls` is in english and `vitesse` in french.  Write everything in english ? (i.e `Speed controls`)